e2e/raw_bootable_test.go

```diff
@@ -38,11 +40,50 @@ var _ = Describe("raw bootable artifacts", Label("raw-bootable"), func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 	It("Should boot as expected", func() {
+		// At first raw images boot on recovery and they reset the system and creates the partitions
+		// so it can take a while to boot in the active partition
+		// lets wait a bit checking
+		By("Waiting for recovery reset to finish", func() {
+			Eventually(func() string {
+				output, _ := vm.Sudo("kairos-agent state")
+				return output
+			}, 5*time.Minute, 1*time.Second).Should(
+				Or(
+					ContainSubstring("active_boot"),
+				))
+		})
+
 		// This checks both that the disk is bootable and with secureboot enabled
-		By("Have secureboot enabled", func() {
-			output, err := vm.Sudo("dmesg | grep -i secure")
-			Expect(err).ToNot(HaveOccurred(), output)
-			Expect(output).To(ContainSubstring("Secure boot enabled"))
+		if os.Getenv("SECUREBOOT") == "true" {
+			By("Have secureboot enabled", func() {
+				output, err := vm.Sudo("dmesg | grep -i secure")
```

Comment:
Can we check for a bit more text than just "secure" ? `dmesg` produces lots of text. On my system for example:

```
~ $ sudo dmesg | grep secure
[79659.762609] intel-ipu6 0000:00:05.0: IPU6 in secure mode
[426107.139126] intel-ipu6 0000:00:05.0: IPU6 in secure mode
[513950.074595] intel-ipu6 0000:00:05.0: IPU6 in secure mode
[600414.944327] intel-ipu6 0000:00:05.0: IPU6 in secure mode
```
(probably nothing to do with secure boot)

Reply:
ummm good point, I did not stole this from kairos so the implementation is worng.

Plus this will also match "secureboot disabled" LOL

Now watch me change it and it starting to fail....

Reply:
ahh you rascal, almost got me!

This only gets the output that matches secure but then 2 lines undernmeath we match against the proper string!

```
Expect(output).To(ContainSubstring("Secure boot enabled"))
```

:D